ocd-1001.sql

```diff
@@ -4,18 +4,30 @@ TRUNCATE openchpl.surveillance CASCADE;
 -- Build temp staging tables for each destination table
 DROP TABLE IF EXISTS surveillance;
```

Comment:
Based on the fact that everything worked last time (except for the function name), no they do not. I'd rather not change something that's not broken.